src/cephadm/cephadm

```diff
@@ -7394,7 +7394,7 @@ class Packager(object):
 
     def repo_gpgkey(self) -> Tuple[str, str]:
         if self.ctx.gpg_url:
-            return self.ctx.gpg_url
+            return self.ctx.gpg_url, 'manual'
```

Comment:
do you think we should also add `manual` to https://github.com/ceph/ceph/blob/9a8f20f42e0fd5753811fefea644d059befb458d/src/cephadm/cephadm#L7683 Otherwise this key won't get cleaned up when removing the repo

Reply:
you are right, it would be better to clean it. I added this in this PR (https://github.com/ceph/ceph/pull/47338/commits/bf0951353a80dc17edcc85a7914cfaf2b956d544)